Python/bytecodes.c

```diff
@@ -5293,6 +5293,32 @@ dummy_func(
             value = PyStackRef_FromPyObjectBorrow(ptr);
         }
 
+        tier2 pure op(_POP_TWO, (pop1, pop2 --)) {
+            PyStackRef_CLOSE(pop2);
+            PyStackRef_CLOSE(pop1);
+        }
+
+        tier2 pure op(_POP_THREE, (pop1, pop2, pop3 --)) {
+            PyStackRef_CLOSE(pop3);
+            PyStackRef_CLOSE(pop2);
+            PyStackRef_CLOSE(pop1);
+        }
+
+        tier2 pure op(_POP_CALL_LOAD_CONST_INLINE_BORROW, (ptr/4, callable, null -- value)) {
```

Comment:
Can you remove the `pure` annotations from these please? They don't get optimized to anything so there's not really a use for them.